flow-typed/platform.js

```diff
@@ -63,6 +63,8 @@ declare type HapiRequest = {
 	},
 	[string]: any,
 };
+
+/*
```

Comment:
ToDo: I need to create a new flow type for `HapiResponseToolkit` https://hapijs.com/api#response-toolkit